netbox/extras/middleware.py

```diff
@@ -22,7 +26,7 @@ def mark_object_changed(instance, **kwargs):
     We have to wait until the *end* of the request to the serialize the object, because related fields like tags and
     custom fields have not yet been updated when the post_save signal is emitted.
     """
-    if not hasattr(instance, 'log_change'):
+    if hasattr(instance, 'log_change') and instance.__class__._meta.verbose_name not in WEBHOOK_MODELS:
```

Comment:
This breaks change logging for any models not in `WEBHOOK_MODELS`. Instead, we can just remove this and add all objects to `_thread_locals.changed_objects` since we're checking for `log_change` after rendering the response.